### ISSUE TITLE
SeqWare Archetype

### ISSUE THREAD

**Issue by briandoconnor** [SeqWare/seqware#307]
Would be great to have the template used for "mvn archetype:generate" for a Java workflow also create a Dockerfile in the HelloWorld example that's created.  This will help people not only make a new workflow but easily integrate it with a docker image.  This opens up the possibility of eliminating our AMI and OVA and just relying on a Docker container for development.

**Comment by denis-yuen** [comment]
I agree. If the DEWrapper workflow takes off in production, we should add a docker workflow of workflows as an example archetype
